src/guide-rngs.md

```diff
@@ -82,8 +71,7 @@ Any block cipher can be turned into a CSPRNG by encrypting a counter. Stream
 ciphers are basically a CSPRNG and a combining operation, usually XOR. This
 means that we can easily use any stream cipher as a CSPRNG.
 
-This crate currently provides two CSPRNGs. The sub-crate `rand_isaac`
-provides two CSPRNG-like PRNGs:
+We provide the following "CSPRNG-like" RNGs:
```

Comment:
CSPRNG-like is a weird word. What does it mean?

Reply:
The rationale for this weird phrasing was to avoid having to throw ISAAC out, though I suppose we could do that, or just put everything in one big table.

This whole section is far from ideal; mostly it's a copy+paste since I didn't want to do a re-write.

Reply:
I would just call ISAAC a CSPRNG. CSPRNG-like does not make sense to me without definition.